addons/website/static/src/js/content/redirect.js

```diff
@@ -30,6 +30,15 @@ document.addEventListener('DOMContentLoaded', () => {
         const backendEditBtnEl = document.querySelector('.o_frontend_to_backend_edit_btn');
         if (backendEditBtnEl) {
             backendEditBtnEl.href = currentUrl.href;
+            document.addEventListener("keydown", ev => {
+                if (
+                    ev.key === "a"
+                    && ev.altKey
+                ) {
```

Comment:
Why not on one line? :thinking:

Reply:
Condition was copy pasted from a longer multiline condition, I kept only what I needed and it didn't bugged me at the time of writing it, but now that you mention it, indeed :o
Changed